system-test/spanner.ts

```diff
@@ -4876,6 +4877,25 @@ describe('Spanner', () => {
         queryStreamMode(done, PG_DATABASE, query, POSTGRESQL_EXPECTED_ROW);
       });
 
+      // test for BATCH WRITE
```

Comment:
nit: No need for the comment. Test name should be self explanatory . Change the test name to

`GOOGLE_STANDARD_SQL should execute mutation group using Batch write'